doc/api/path.md

```diff
@@ -241,82 +333,89 @@ path.parse('/home/user/dir/file.txt')
 // {
 //    root : "C:\\",
 //    dir : "C:\\path\\dir",
-//    base : "index.html",
-//    ext : ".html",
-//    name : "index"
+//    base : "file.txt",
+//    ext : ".txt",
+//    name : "file"
 // }
 ```
 
+```text
+┌─────────────────────┬────────────┐
+│          dir        │    base    │
+├──────┬              ├──────┬─────┤
+│ root │              │ name │ ext │
+" C:\      path\dir   \ file  .txt "
+└──────┴──────────────┴──────┴─────┘
+(all spaces in the "" line should be ignored -- they're purely for formatting)
+```
+
+A [`TypeError`][] is thrown if `path` is not a string.
+
 ## path.posix
 
-Provide access to aforementioned `path` methods but always interact in a posix
-compatible way.
+The `path.posix' property provides access to POSIX specific implementations
+of the `path` methods.
 
 ## path.relative(from, to)
 
-Solve the relative path from `from` to `to`. `from` and `to` must be strings.
+* `from` {String} A file path
+* `to` {String} A file path
 
-At times we have two absolute paths, and we need to derive the relative
-path from one to the other.  This is actually the reverse transform of
-`path.resolve`, which means we see that:
+The `path.relative()` method returns the relative path from `from` to `to`.
+If `from` and `to` are equal, a zero-length string is returned.
+
+If a zero-length string is passed as `from` or `to`, the current working
+directory will be used instead of the zero-length strings.
+
+For example on POSIX:
 
 ```js
-path.resolve(from, path.relative(from, to)) == path.resolve(to)
+path.relative('/data/orandea/test/aaa', '/data/orandea/impl/bbb')
+// returns '../../impl/bbb'
 ```
 
-Examples:
+On Windows:
 
 ```js
 path.relative('C:\\orandea\\test\\aaa', 'C:\\orandea\\impl\\bbb')
 // returns '..\\..\\impl\\bbb'
-
-path.relative('/data/orandea/test/aaa', '/data/orandea/impl/bbb')
-// returns '../../impl/bbb'
 ```
 
-*Note:* If the arguments to `relative` have zero-length strings then the current
-        working directory will be used instead of the zero-length strings. If
-        both the paths are the same then a zero-length string will be returned.
+A [`TypeError`][] is thrown if neither `from` or `to` is a string.
 
 ## path.resolve([from ...], to)
 
-Resolves `to` to an absolute path. All arguments must be strings.
+* `[from ...]` {String} One or more file paths
+* `to` {String} A file path
 
-If `to` isn't already absolute `from` arguments are prepended in right to left
-order, until an absolute path is found. If after using all `from` paths still
-no absolute path is found, the current working directory is used as well. The
-resulting path is normalized, and trailing slashes are removed unless the path
-gets resolved to the root directory. Empty string `from` arguments are
-ignored.
+The `path.resolve()` method resolves `to` to an absolute path.
 
-Another way to think of it is as a sequence of `cd` commands in a shell.
+If `to` is not already an absolute path, each `from` argument is prepended in
```

Comment:
Perhaps this would be simpler to explain if we described all of the parameters the same, since the implementation does not treat what is currently labeled as `to` any different than any of the `from` arguments.